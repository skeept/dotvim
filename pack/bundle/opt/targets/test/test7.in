x "a" "b" c
x "a" "b" c
x "a" "b" c
a "b" "c" x "d" "e" f
a "b" "c" x "d" "e" f
a "b" "c" x "d" "e" f
a "b" "c" x "d" "e" f
a "b" "c" x "d" "e" f
a "b" "c" "x" "d" "e" f
a "b" "c" "x" "d" "e" f
a "b" "c" "x" "d" "e" f
a "b" "c" "x" "d" "e" f
a "b" "c" "x" "d" "e" f
a "b" "c" x
a "b" "c" x
a "b" "c" x
