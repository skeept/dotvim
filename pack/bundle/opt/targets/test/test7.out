x "A" "b" c
x "A" "b" c
x "a" "B" c
a "b" "c" x "D" "e" f
a "b" "c" x "D" "e" f
a "b" "c" x "d" "E" f
a "b" "C" x "d" "e" f
a "B" "c" x "d" "e" f
a "b" "c" "X" "d" "e" f
a "b" "c" "x" "D" "e" f
a "b" "c" "x" "d" "E" f
a "b" "C" "x" "d" "e" f
a "B" "c" "x" "d" "e" f
a "b" "C" x
a "b" "C" x
a "B" "c" x
